Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -24,6 +24,10 @@ @interface SessionDelegate : NSObject <ARSessionDelegate, MTKViewDelegate>
         bool operator()(const ARPlaneAnchor* lhs, const ARPlaneAnchor* rhs) const {
             return lhs.identifier < rhs.identifier;
         }
+        API_AVAILABLE(ios(13.4))
+        bool operator()(const ARMeshAnchor* lhs, const ARMeshAnchor* rhs) const {
+            return lhs.identifier < rhs.identifier;
+        }
```

Comment:
If this code is compiled with an older version of XCode, which uses an older version of the iOS SDK, then the `ARMeshAnchor` type is unknown. The CI builds this code with both older and newer versions of XCode, and when using newer versions of XCode also builds with different target iOS versions. To handle the case where we are compiling with an older version of XCode/iOS SDK, I think we also need to conditionally remove this from even being compiled. I think something like:
```suggestion
#if (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_4)
        API_AVAILABLE(ios(13.4))
        bool operator()(const ARMeshAnchor* lhs, const ARMeshAnchor* rhs) const {
            return lhs.identifier < rhs.identifier;
        }
#endif
```